### PR TITLE
[recipe] mul_cn and unified char and bpe data format

### PR DESCRIPTION
**Pull request by whiteshirt0429**

### CODE REVIEW

File: examples/multi_cn/s0/run.sh [hunk 1]
```diff
@@ -0,0 +1,292 @@
+#!/bin/bash
+
+# Copyright 2021 JD AI Lab. All Rights Reserved. (authors: Lu Fan)
+# Copyright 2021 Mobvoi Inc. All Rights Reserved.
+# Author: di.wu@mobvoi.com (DI WU)
```
Comment (robin1001) [inline examples/multi_cn/s0/run.sh:5]: Copyright 2021 Mobvoi Inc. All Rights Reserved. (Di Wu)

File: examples/multi_cn/s0/run.sh [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+        sed -i 's/\xEF\xBB\xBF//' ${feat_dir}_${en_modeling_unit}/test_${x}/text
+    done
+
+    tools/compute_cmvn_stats.py --num_workers 16 --train_config $train_config \
+        --in_scp data/${train_set}/wav.scp \
+        --out_cmvn ${feat_dir}_${en_modeling_unit}/$train_set/global_cmvn
+
+fi
+
+bpecode=../../librispeech/s0/data/lang_char/train_960_unigram5000.model
```
Comment (robin1001) [inline examples/multi_cn/s0/run.sh:145]: 呃 what's this? it shouldn't be here.

File: examples/multi_cn/s0/run.sh [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+
+# Copyright 2021 JD AI Lab. All Rights Reserved. (authors: Lu Fan)
+# Copyright 2021 Mobvoi Inc. All Rights Reserved.
+# Author: di.wu@mobvoi.com (DI WU)
+. ./path.sh || exit 1;
+
+# Use this to control how many gpu you use, It's 1-gpu training if you specify
+# just 1gpu, otherwise it's is multiple gpu training based on DDP in pytorch
+export CUDA_VISIBLE_DEVICES="0,1,2,3"
+stage=0 # start from 0 if you need to start from data preparation
```
Comment (robin1001) [inline examples/multi_cn/s0/run.sh:11]: maybe add multiple machine training support here.

File: examples/multi_cn/s0/tools [hunk 4]
```diff
@@ -0,0 +1 @@
+../../aishell/s0/tools/
```
Comment (robin1001) [inline examples/multi_cn/s0/tools:1]: ../../../tools/

**Comment by fanlu** [comment]
https://github.com/kaldi-asr/kaldi/pull/4433
maybe this pr is useful for speed up stcmds data prepare

**Review by robin1001 — CHANGES_REQUESTED** [review]
please see inline, and please also check in the result.

File: examples/multi_cn/s0/README.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
+# Performance Record
+
+This is a Chinese speech recognition recipe that trains on all Chinese corpora on [OpenSLR](http://www.openslr.org), including:
+* Aidatatang (140 hours)
```
Comment (robin1001) [inline examples/multi_cn/s0/README.md:4]: on all open source Chinese corpus, aishell2 and TAL are not on OpenSLR.

File: examples/multi_cn/s0/README.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+* ST-CMDS (110 hours)
+* THCHS-30 (26 hours)
+
+* Feature info: using fbank feature, with cmvn, no speed perturb.
+* Training info: lr 0.004, batch size 18, 3 machines, 3*8 = 24 gpu, acc_grad 1, 220 epochs, dither 0.1
+* Decoding info: ctc_weight 0.5, average_num 30
+* Git hash: 013794572a55c7d0dbea23a66106ccf3e5d3b8d4
+* Model link: http://mobvoi-speech-public.ufile.ucloud.cn/public/wenet/mul_cn/20210315_unified_transformer_exp.tar.gz
+
+aishell-1 results
```
Comment (robin1001) [inline examples/multi_cn/s0/README.md:28]: it's better is we list all results in just one table.

**Review by robin1001 — CHANGES_REQUESTED** [review]
please see inline.

**Review by robin1001 — APPROVED** [review]

**Pull request merged**